pkg/destroy/gcp/network.go

```diff
@@ -712,23 +848,39 @@ func (o *ClusterUninstaller) destroyCloudControllerInternalLBs() error {
 		return err
 	}
 
+	errs := []error{}
+	found := make([]string, 0, len(backends))
```

Comment:
nit: the order of errs: and found: is different from everywhere else in the code.